drivers/cime/esm.F90

```diff
@@ -655,18 +653,13 @@ subroutine AddAttributes(gcomp, driver, config, compid, compname, inst_suffix, n
     if (chkerr(rc,__LINE__,u_FILE_u)) return
 
     !------
-    ! Add restart flag a to gcomp attributes
+    ! Add driver restart flag a to gcomp attributes
     !------
     attribute = 'read_restart'
```

Comment:
it seems this is changing the attributes that controls reading restart file? Does it requires also changes in the component side (I mean cap)?

Reply:
There are not changes required in the component caps.